images/plbase/python-requirements.txt

```diff
@@ -43,3 +43,5 @@ types-Pygments==2.15.0.1
 types-pytest-lazy-fixture==0.6.3.3
 typing-extensions==4.7.0
 tzlocal==4.3
+schemdraw==0.17
+*Add schemdraw:0.17 to PrairieLearn
```

Comment:
Why is this line here?

Reply:
https://prairielearn.readthedocs.io/en/latest/questionRuntime/#installing-libraries-in-your-course

This line told me to leave a comment on the request... was it the wrong file to leave comment?

Reply:
If you're talking about to "At the bottom of the page you can give a descriptive title to your changes", that's referring to the commit message. There should not be any additional changes to the file itself. In this case, you could have replaced "Update python-requirements.txt".

I can see how "at the bottom of the page" is confusing given that GitHub recently changed the in-browser editor for files. I can update those docs to make this clearer for future contributors.

Reply:
Updated docs: https://github.com/PrairieLearn/PrairieLearn/pull/8125

Reply:
Thank you very much!